src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,30 @@
 import React from 'react';
+import { Comment } from '../../Types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => {
+  const { name, email, body } = comment;
```

Comment:
If we destructure 3 or more properties, it would be better to move every property to a new line
```suggestion
  const {
    name,
    email,
    body,
  } = comment;
```